### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by emanuelespies**
What do you think of this version?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
```
Comment (isadoramuller) [inline .github/pull_request_template.md]: we wont perform any test. Please delete.

**Review by isadoramuller — CHANGES_REQUESTED** [review]
ok

File: .github/pull_request_template.md (hunk 1 above)
Comment (emanuelespies) [reply .github/pull_request_template.md]: Ok. Will fix

**Review by isadoramuller — APPROVED** [review]
LGTM :D

**Pull request merged**